app/const.py

```diff
@@ -305,4 +305,5 @@
 
 # The threshold for email notification to an admin user. When the number of
```

Comment:
This is comment for DEFAULT_UNREVIEWED_NOTES_EMAIL_THRESHOLD, so move this to just before DEFAULT_UNREVIEWED_NOTES_EMAIL_THRESHOLD.

Optional: You may remove DEFAULT_UNREVIEWED_NOTES_EMAIL constant and use '' directly.